chunky/src/java/se/llbit/chunky/ui/SceneChooserController.java

```diff
@@ -173,7 +178,11 @@ public void setStage(Stage stage) {
   private void populateSceneTable(File sceneDir) {
     List<SceneListItem> scenes = new ArrayList<>();
     List<File> fileList = SceneHelper.getAvailableSceneFiles(sceneDir);
-    Collections.sort(fileList);
+    // TODO: Sort by recently modified instead of file name.
+    Collections.sort(fileList, Comparator
+            .comparing((File o) -> -(Long)o.lastModified()) // negative of last modified -> most recent at top
+            .thenComparing((File o) -> o.toString().toLowerCase()) // Then by file name, ignoring case.
+            .thenComparing(o -> o));
```

Comment:
This line does nothing?

Reply:
If a scene is duplicated by copy and paste, they could have the same modify time, but not the same file name, and thus a thenComparing will ensure that it is always sorted in the same way (similar to how minecraft does it), otherwise they might get sorted into different orders on different reads.

Reply:
Yes, but the third compare still doesn't do anything, does it?

Reply:
The third comparison only happens as fallback when lowercase pathnames are equal (depending on OS/file system) and checks `File::compareTo` via `Comparable` interface:
https://docs.oracle.com/javase/8/docs/api/java/io/File.html#compareTo-java.io.File-
_Compares two abstract pathnames lexicographically. The ordering defined by this method depends upon the underlying system._

But because this case is rarely relevant, we could remove the second comparison entirely and simply use the last one for both.

Reply:
I've added my suggestion for a (imo) easier understandable comparator (without the third comparison). :)

Reply:
Comparison of sortings: https://gist.github.com/ShirleyNekoDev/1c3f58a9caa9e1943bac2b7cb46948bd
If you prefer your current sort oder I won't object ;)